services/web/server/tests/sandbox/projects_openapi_generator.py

```diff
@@ -55,10 +138,39 @@ class Node(BaseModel):
     # var outputs?
 
 
+# --------------
 class Project(BaseModel):
+    """Domain model"""
+
     id: UUID
     pipeline: Dict[UUID, Node]
 
+
+# requests models
+class ProjectNew(BaseModel):
+    pipeline: Dict[UUID, Node]
+
+
+class ProjectUpdate(BaseModel):
+    # same as new but ALL optional??
+    # some validators?
+    pass
+
+
+# response models
+class ProjectItem(BaseModel):
+    # Lightweight and part of an array
+    id: UUID
+
+    url: HttpUrl
+
+
+class ProjectDetail(BaseModel):
+    id: UUID
```

Comment:
same and be careful with this name. cause I think in the database we actually have both id and uuid, the first one being an integer and the second a uuid